source/concepts/block.rst

```diff
@@ -20,6 +20,12 @@ Block creation
 
 Blocks are created by :doc:`accounts <account>`. The process of creating new blocks is called :doc:`harvesting <harvesting>`. The harvesting account - called the harvester - gets the fees for the transactions in the block. This gives the harvester an incentive to add as many transactions to the block as possible.
 
+****************
+Block Difficulty
+****************
+
+The difficulty for a new block is calculated from the difficulties and time stamps of the last 60  blocks such that the new difficulty is equal to the average difficulty divided by the average block creation time for the last 60 blocks. If the new difficulty is more than 5% greater or smaller than the difficulty of the last block, then the change is capped to 5%. Additionally, difficulties are kept within certain bounds. The new difficulty is clamped to the boundaries if it is greater than 10\ :sup:`15` or smaller than 10\ :sup:`13`\ .  The algorithm produces approximately one block every 15 seconds.
```

Comment:
I'd merge the sentence into one:

``The difficulty for a new block is equal to the average difficulty divided by the average block creation time for the last ``60`` blocks. ``

Also, I'd remove the following sentence, since the block creation paragraph already talks about it.

``The algorithm produces approximately one block every 15 seconds.``

Reply:
Do you know which are the boundaries? If not we can ask.

Reply:
> Do you know which are the boundaries? If not we can ask.

The boundaries I used came from the February 2018 tech ref.

Reply:
> `The difficulty for a new block is equal to the average difficulty divided by the average block creation time for the last `60`blocks.`
> 
> Also, I'd remove the following sentence, since the block creation paragraph already talks about it.
>
>You are right it. 60 should be between double "`" symbols.

I made these adjustments and resubmit my pull request. If my other comments beget additional edits, I will make them then.